crimsobot/cogs/mystery.py

```diff
@@ -28,6 +28,27 @@ async def tarot(self, ctx: commands.Context) -> None:
         # if no subcommand provided, give a three-card reading
         await self.ppf(ctx)
 
+    @tarot.command(name='one', brief='Get a single card.')  # something better needs to be put here, i do not know much
+    @commands.cooldown(3, 300, commands.BucketType.user)
+    async def one(self, ctx: commands.Context, spread: str = 'one') -> None:
+        """ One card """  # this also needs to be improved
+
+        fp, descriptions = await tarot.reading(spread)
+        filename = 'reading.png'
```

Comment:
This seems wasteful.

Reply:
what do you mean by wasteful?

Reply:
Unneeded assignment - a string literal could be used in `f = discord.File(fp, filename)` (so that it becomes `f = discord.File(fp, 'reading.png')`) without issue.

Reply:
Filename is defined separately to provide this as the embed description as well.